d3d9ex/AutoFix.cpp

```diff
@@ -241,7 +241,7 @@ void MainContext::FF13_InitializeGameAddresses()
 {
 	// FF13 always seem to use the same addresses (even if you force ASLR on the OS), but we are calculating the addresses just in case...
 	uint8_t* baseAddr = (uint8_t*)GetModuleHandle(NULL); // Should be 400000
-	PrintLog("Base Addr = %x", baseAddr);
+	spdlog::info("Base Addr = {:X}", baseAddr);

```

Comment:
This is not logging (needs to use `{}` and cast to `void*`?)
FF13-2 equivalent line needs the same change.

It is unfortunate that it simply omits the log message and it doesn't throw a compilation warning or error

Reply:
That was because of wrong format specifier, yes, cast to `void*` and `{p`} (which is auto guessed by `{}`) is correct.